internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -633,7 +633,10 @@ func (r *Reconciler) CompositeReconcilerOptions(ctx context.Context, d *v1.Compo
 		}
 
 		h := EnqueueCompositeResources(resource.CompositeKind(d.GetCompositeGroupVersionKind()), r.engine.GetCached(), r.log)
-		o = append(o, composite.WithWatchStarter(composite.ControllerName(d.GetName()), h, r.engine))
+		o = append(o,
+			composite.WithWatchStarter(composite.ControllerName(d.GetName()), h, r.engine),
+			composite.WithPollInterval(0), // Disable polling.
```

Comment:
@turkenh You can see on line 551 of this file that it'll default to `composite.WithPollInterval(r.options.PollInterval)`.

We only disable polling here inside the `if r.options.Features.Enabled(features.EnableBetaRealtimeCompositions)` branch.